sysml.library/Kernel Library/Objects.kerml

```diff
@@ -63,8 +63,8 @@ package Objects {
 	 * to be the same.
 	 */
 	assoc SelfLink specializes BinaryLink {
-		end feature self: Anything[0..*] nonunique redefines source subsets myself redefines Anything::self;
-		end feature myself: Anything[0..*] nonunique redefines target subsets self;
+		end feature self: Anything[1..1] nonunique redefines source subsets myself redefines Anything::self;
```

Comment:
Can we make these unique?  I know it doesn't matter for [1..1], but it's a bit distracting and has implications (for future language definers) if this were ever changed to multi-valued.

Reply:
Yes, I meant to remove the `nonunique` on these lines.